CHANGELOG.md

```diff
@@ -116,6 +116,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Clean up and rebuild `@osd/pm` ([#3570](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3570))
 - [Vega] Add Filter custom label for opensearchDashboardsAddFilter ([#3640](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3640))
 - [Timeline] Fix y-axis label color in dark mode ([#3698](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3698))
+- [Dashboard] Fix Dashboard-Delete from Dashboards / Dashboards (broken by #3090) ([#3796](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3796))
```

Comment:
Reading this, with no context, I am not sure what the change is or is trying to fix. Can we reword this please?

Reply:
There's no need for a changelog entry here, as this bug was only introduced in a 2.7 feature.